app/views/amusement_parks/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Welcome to: <%= @parks.name %></h1>
+
+
+<h3>Admission Cost:<%= @parks.admission_cost %>
+
+  <ul>
+  <% @mechanics_on_site.each do |mechanic| %>
```

Comment:
What was `@mechanics_on_site` returning for you? 

If you see my feedback on your `amusement_park_controller.rb`, you would want the method that returns all of the `mechanics` in your `amusement_park.rb` model. Then in here you would call that method on your `@parks`. 
```ruby
@parks.mechanics_on_site.each do |mechanic|
```